bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/Utils.java

```diff
@@ -91,4 +102,25 @@ public static JsonObject convertFileToJSON(URL fileName) throws JsonIOException,
         jsonObject = jsonElement.getAsJsonObject();
         return jsonObject;
     }
+
+    @NonNullByDefault
+    public List<URL> findDatabaseFiles(Logger logger) {
+        List<URL> urlEntries = new ArrayList<>();
+        try {
+            File[] userDbFiles = new File(ConfigConstants.getUserDataFolder() + File.separator + BINDING_ID)
```

Comment:
I'm wondering if qe better put that in the conf directory.

Reply:
The reasoning to put it in userdata folder was that 1) I would not know an appropriate folder in the conf directory 2) Other bindings, e.g. zwave, mqtt, openhabcloudvoicerss etc use the userdata folder.

I'm very willing to change if you think it should be the conf folder, in that case pls advice which of the subfolders I should take.

Reply:
@J-N-K let me know if you want to have this changed.
As we have quite some new devices arriving, I'm keen to have this in place so others can much easier get their devices going and we can add them to the binding once they have a working config for their device.

Reply:
@J-N-K Please let me know your advice.
I'm quite keen on getting this PR processed as we have a lot of new devices coming. With this change users can test their files and we can include tested files in the binding and have better supported new devices